Polluted
Traceback (most recent call last):
  File "/global/home/users/kalatimer/proj/20supersat/src/wrf/dsd_q1_vs_q4_figsrc.py", line 161, in <module>
    main()
  File "/global/home/users/kalatimer/proj/20supersat/src/wrf/dsd_q1_vs_q4_figsrc.py", line 46, in main
    cutoff_bins, full_ss, incl_rain, incl_vent)
  File "/global/home/users/kalatimer/proj/20supersat/src/wrf/dsd_q1_vs_q4_figsrc.py", line 60, in make_and_save_dsd_q1_vs_q4
    'wrfout_d01_all_dsdsum_vars_v2', 'r')
  File "netCDF4/_netCDF4.pyx", line 2135, in netCDF4._netCDF4.Dataset.__init__
  File "netCDF4/_netCDF4.pyx", line 1752, in netCDF4._netCDF4._ensure_nc_success
FileNotFoundError: [Errno 2] No such file or directory: b'/clusterfs/stratus/dromps/wrf_fan2018/Supersaturation/Supersaturation/C_BG/wrfout_d01_all_dsdsum_vars_v2'
